framework/src/modules/chain/logic/block.js

```diff
@@ -57,16 +57,14 @@ class Block {
 	}
 
 	/**
-	 * Sorts input data transactions.
-	 * Calculates reward based on previous block data.
-	 * Generates new block.
+	 * Sorts transactions for later including in the block.
 	 *
-	 * @param {Object} data
-	 * @returns {block} block
-	 * @todo Add description for the params
+	 * @param {Array} transactions Unsorted collection of transactions
+	 * @returns {Array} transactions Sorted collection of transactions
+	 * @static
 	 */
-	create(data) {
-		const transactions = data.transactions.sort((a, b) => {
+	static sortTransactions(transactions) {
```

Comment:
do we need to change this now?
Prefer to keep it as it was if it's unnecessary for the conflict

Reply:
Yes we need to extract that logic from deep nested blocks, to verify the integrity of blocks in integration tests.